jekyll/_cci2/contexts.md

```diff
@@ -237,121 +237,110 @@ to set up your CircleCI command line interface._
 
-3. Replace the existing environment variable under that context. Execute this
-   command in the CLI and replace the `secret name` with the name of the
-   environment variable from Step 2: \
-  `circleci context store-secret <vcs-type> <org-name> <context-name> <secret name>`
+3. Update the existing environment variable under that context. Execute this
+   command in the CLI and replace the `env-var-name` with the name of the
+   environment variable from Step 2:
+  `circleci context store-secret <vcs-type> <org-name> <context-name> <env-var-name>`
 
+Note that the CLI will prompt you to input the secret value, rather than accepting it 
+as an argument. This approach is designed to avoid unintentional secret exposure.
 
 ##### Using CircleCI’s API
 {:.no_toc}
 
-To rotate an environment variable from our API, perform the following steps:
-
-1. Call the [Update Environment Variable](https://circleci.com/docs/api/v2/#operation/addEnvironmentVariableToContext)
-   endpoint with the appropriate request body in one of our supported languages.
-   For this request, replace the `context-id` and the `env-var-name` with the ID
-   for the context and the environment variable name that should be updated. The
-   request body should include a `value` key with a string value assigned.
-
-   To see request samples in all supported languages, visit our [api docs on
-   updating an environment
-   variable](https://circleci.com/docs/api/v2/#operation/addEnvironmentVariableToContext).
-
-2. If the request is successful, the response will include the name of the
-   environment variable that has been updated. An unsuccessful response will
-   include a clear error message explaining the error reported.
+To rotate an environment variable from our API, call the [Update Environment Variable](https://circleci.com/docs/api/v2/#operation/addEnvironmentVariableToContext)
+endpoint with the appropriate request body. For this request, replace the `context-id` 
+and the `env-var-name` with the ID for the context and the environment variable name 
+that should be updated. The request body should include a `value` key containing the 
+plaintext secret as a string.
 
 
 ## Secrets masking
 _Secrets masking is not currently available on self-hosted installations of CircleCI Server_
 
-Contexts hold project secrets or keys that perform crucial functions for your applications. For added security CircleCI performs secret masking on the build output, obscuring the `echo` or `print` output of contexts.
+Contexts hold potentially sensitive secrets that are not intended to be exposed. For added security, 
+CircleCI performs secret masking on the build output, obscuring `echo` or `print` output that 
+contains env var values.
 
 The value of the context will not be masked in the build output if:
 
-* the value of the context is less than 4 characters
-* the value of the context is equal to one of `true`, `True`, `false` or `False`
```

Comment:
These are env var values _within_ the context, but in this case, it seemed fine to just remove the prepositional bit rather than updating